diff --git a/node_modules/element-ui/packages/date-picker/src/basic/date-table.vue b/node_modules/element-ui/packages/date-picker/src/basic/date-table.vue
index e77ab07..03cfd86 100644
--- a/node_modules/element-ui/packages/date-picker/src/basic/date-table.vue
+++ b/node_modules/element-ui/packages/date-picker/src/basic/date-table.vue
@@ -20,7 +20,7 @@
         v-for="(cell, key) in row"
         :class="getCellClasses(cell)"
         :key="key">
-        <div>
+        <div :data-content="cell.lunar">
           <span>
             {{ cell.text }}
           </span>
@@ -35,6 +35,7 @@
   import { getFirstDayOfMonth, getDayCountOfMonth, getWeekNumber, getStartDateOfMonth, prevDate, nextDate, isDate, clearTime as _clearTime} from 'element-ui/src/utils/date-util';
   import Locale from 'element-ui/src/mixins/locale';
   import { arrayFindIndex, arrayFind, coerceTruthyValueToArray } from 'element-ui/src/utils/util';
+  import dateMethods from 'xe-utils/date'
 
   const WEEKS = ['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat'];
   const getDateTimestamp = function(time) {
@@ -196,6 +197,10 @@
             cell.disabled = typeof disabledDate === 'function' && disabledDate(cellDate);
             cell.selected = arrayFind(selectedDate, date => date.getTime() === cellDate.getTime());
             cell.customClass = typeof cellClassName === 'function' && cellClassName(cellDate);
+            //添加农历
+            console.log('农历',cellDate)
+            cell.lunar = '农历'
+            cell.lunar = LunarCalendar.solarToLunar(cellDate.getFullYear(), cellDate.getMonth() + 1, cellDate.getDate()).lunarDayName
             this.$set(row, this.showWeekNumber ? j + 1 : j, cell);
           }
 
